### ISSUE TITLE
Date conversion wrong - does use localtimezone when storing data in database

### ISSUE THREAD

**Issue by sfakir** [balderdashy/sails-mysql#248]
Model.create({
  time: new Date('Sun Aug 30 2015 17:35:00 GMT+0200 (CEST)')
  time2: new Date('2015-08-30T15:35:00.000Z')
});

both of them should be 15:35 in database. But it stores 17:35 in both cases..  (local timezone GMT +02:00). Settings has UTC enabled:

  localhostSQL: {
    adapter: 'sails-mysql',
    host: 'localhost',
    user: 'root',
    password: '',
    database: 'mitfahrerapp',
    timezone: 'utc'

  }

**Comment by sfakir** [comment]
The reason is, that you use the wrong methods in lib/utils.js

It should be  getUTCHours() instead of getHours() - for all the methods.


date = date.getFullYear() + '-' +
    ('00' + (date.getMonth()+1)).slice(-2) + '-' +
    ('00' + date.getDate()).slice(-2) + ' ' +
    ('00' + date.getHours()).slice(-2) + ':' +
    ('00' + date.getMinutes()).slice(-2) + ':' +
    ('00' + date.getSeconds()).slice(-2);

**Comment by sailsbot** [comment]
Thanks for posting, @sfakir.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by particlebanana** [comment]
By default Waterline uses the local dates. In an earlier version we had UTC dates and everyone complained that their dates were not correct. I agree this should probably be an option or just use epoch stamps instead of dates. But in the current version all dates are local time. If you would like to change this you can submit a PR to the waterline roadmap where there can be a discussion.
